docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.8'
 
-services:  
-  
+services:
```

Comment:
Can you also update console and cortex version here

Reply:
Sure thing!